smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/model/SymbolBuilder.kt

```diff
@@ -108,13 +109,6 @@ fun Symbol.isOptional(): Boolean {
     return this.getProperty("isOptional").orElse(false) as Boolean
 }
 
-fun Symbol.toInternalSPI(spiName: String): Symbol {
-    return this.toBuilder()
-        .putProperty("isInternalSPI", true)
-        .putProperty("spiName", spiName)
-        .build()
-}
-
```

Comment:
This is removed, the symbol factory method does this for us now when the symbol is made